<launch>
    <!-- <node pkg="map_server" type="map_server" name="map_server_node" args="$(find extract_globalpath)/map_file/map.yaml"/> -->
    <node pkg="static_map" type="static_map_node" name="static_map_node">
        <param name="pcd_file" value="$(find static_map)/map_file/yunle_car/zetong_vectormap_leaf2.pcd" />
        <param name="map_frame" value="map" />
        <param name="duration" value="1.0" />
        <remap from="/static_map" to="/map/point_cloud" />
    </node>
    <node pkg="rviz" type="rviz" name="Rviz_for_extract_path" args="-d $(find extract_globalpath)/rviz/test.rviz" />
    <node pkg="extract_globalpath" type="extract_whole_node" name="extract_whole_node" output="screen">
        <param name="path_file" value="$(find extract_globalpath)/path_file/path.csv"/>
        <param name="path_prefix" value="$(find extract_globalpath)/path_file_Yunlecar/" />
        <param name="weight_data" value="0.47" />
        <param name="weight_smooth" value="0.2" />
        <param name="tolerance" value="0.1" />
    </node>
</launch>
